caffe2/operators/hip/conv_op_miopen.hip

```diff
@@ -667,4 +774,9 @@ bool MIOPENConvGradientOp::RunOnDevice() {
 
 REGISTER_MIOPEN_OPERATOR(Conv, MIOPENConvOp);
 REGISTER_MIOPEN_OPERATOR(ConvGradient, MIOPENConvGradientOp);
+
+// 3-D Convolutions.
```

Comment:
just remove comment, I think the naming is clear enough, remove empty line before as well

Reply:
Done